README.md

```diff
@@ -1,59 +1,114 @@
+   Depending on the target, you can build the example project with the GCC_ARM, ARM or IAR toolchain. To learn which toolchain supports your target, run this command:
+
+   ```bash
+   $ mbed compile -S
+   ```
+
+1. Clone this repository on your system.
+1. Change the current directory to where the project was cloned.
+
+## Building and running
+
+1. Connect a USB cable between the USB port on the target and the host computer.
+1. Run this command to build the example project and program the microcontroller flash memory:
+
+    ```bash
+    $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm
+    ```
+
+(Note: You can use the Mbed CLI command-line option "--sterm" to open a serial terminal after flashing.)
+
+Your PC may take a few minutes to compile your code.
+
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-thread-statistics.bin`.
+
+Alternatively, you can manually copy the binary to the target, which gets mounted on the host computer through USB.
+
+## Expected output
+
+The serial terminal shows an output similar to the following:
 
 ```
-mbed compile -m K64F -t ARM
-```
+--- Terminal on /dev/ttyACM0 - 9600,8,N,1 ---
+ID: 0x20000e20 
+Name: main 
+State: 2 
+Priority: 24 
+Stack Size: 4096 
+Stack Space: 3896 
+
+ID: 0x20000f70 
+Name: rtx_idle 
+State: 1 
+Priority: 1 
+Stack Size: 512 
+Stack Space: 256 
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
+ID: 0x1fff0580 
+Name: blinky_thread 
+State: 3 
+Priority: 24 
+Stack Size: 224 
+Stack Space: 128 
+
+ID: 0x20000f2c 
+Name: rtx_timer 
+State: 3 
+Priority: 40 
+Stack Size: 768 
+Stack Space: 664 
+
+ID: 0x1fff0730 
+Name: idle_thread 
+State: 3 
+Priority: 24 
+Stack Size: 384 
+Stack Space: 280 
+```
+The information below shows how to interpret the above fields:
 
 ```
-[snip]
-Elf2Bin: mbed-os-example-thread-statistics
-+------------------+-------+-------+------+
-| Module           | .text | .data | .bss |
-+------------------+-------+-------+------+
-| [lib]\c_w.l      | 11473 |    16 |  348 |
-| [lib]\cpprt_w.l  |    36 |     0 |    0 |
-| [lib]\fz_wm.l    |    18 |     0 |    0 |
-| [lib]\m_wm.l     |    48 |     0 |    0 |
-| anon$$obj.o      |    32 |     0 | 1024 |
-| main.o           |   196 |     0 |    0 |
-| mbed-os\drivers  |   130 |     0 |    0 |
-| mbed-os\features |   132 |     0 |  304 |
-| mbed-os\hal      |  1660 |    30 |   64 |
-| mbed-os\platform |  3657 |   104 |  604 |
-| mbed-os\rtos     | 13822 |  2310 | 4592 |
-| mbed-os\targets  |  9193 |   104 |  324 |
-| Subtotals        | 40397 |  2564 | 7260 |
-+------------------+-------+-------+------+
-Total Static RAM memory (data + bss): 9824 bytes
-Total Flash memory (text + data): 42961 bytes
+ID:            Thread id
+Name:          Thread name
+State:         Thread states:
+               osThreadInactive   =  0,
```

Comment:
`State` is an unsigned integer, so you should just say `0: Thread is inactive` etc.

Reply:
The headline is `Thread states`,  so we keep state alone as looks below:
```
State:         Thread states:
               0: Inactive,
               1: Ready,
               2: Running,
               3: Blocked,
               4: Terminated,
              -1: Error  
```